sw/host/opentitanlib/src/app/config/structs.rs

```diff
@@ -145,6 +154,20 @@ pub struct I2cConfiguration {
     pub alias_of: Option<String>,
 }
 
+/// Configuration of a particular peripheral.
+#[derive(Default, Deserialize, Clone, Debug)]
+pub struct IoPeripheral {
+    /// The user-visible name of this pheripheral.
+    pub name: String,
+    /// List of pins that this peripheral is connected to.
+    pub pins: Vec<String>,
+    /// If present, this peripheral will only be enable if that particular pin is high.
+    pub enable_pin: Option<String>,
+    /// If set, the enable signal is inverted (i.e. active low).
+    #[serde(default)]
+    pub enable_low: bool,
```

Comment:
bikeshedding:
```suggestion
    pub enable_invert: bool,
```

Reply:
The concept of inverted signal already exists in https://github.com/lowRISC/opentitan/blob/cbca064766066e90c06addfbcc7d2ee465d55ca7/sw/host/opentitanlib/src/app/config/opentitan_cw340.json#L8